CHANGES/7740.bugfix

```diff
@@ -0,0 +1 @@
+Sync all Collection versions
```

Comment:
A better changelog message here would be great.

Reply:
How does it sound:
```
Enable sync for more than 10 collection versions
```